br/pkg/lightning/mydump/loader.go

```diff
@@ -517,12 +517,30 @@ func (s *mdLoaderSetup) route() error {
 		}
 	}
 	for _, info := range s.tableSchemas {
+		if _, ok := knownDBNames[info.TableName.Schema]; !ok {
+			knownDBNames[info.TableName.Schema] = &dbInfo{
+				fileMeta: info.FileMeta,
+				count:    1,
```

Comment:
I'm not sure we should use 1 or 0 here. Wrong value may cause problem in following logic. Can you add an integration test for it?